mkdocs.yml

```diff
@@ -101,6 +102,11 @@ extra_css:
   - stylesheets/color.css
   - stylesheets/fonts.css
 
+extra_javascript:
+  - assets/js/mathjax.js
+  - https://polyfill.io/v3/polyfill.min.js?features=es6
```

Comment:
no external js for privacy reasons ;)

Reply:
why do we need polyfill?

Reply:
Oh I see! Apparently it is needed to enable [mathjax](https://squidfunk.github.io/mkdocs-material/reference/mathjax/). In other words to enable LaTeX math notation.

Reply:
Okay, so we need to use the external sources. I think the two sources can be considered trustworthy.